foo_uie_console/main.cpp

```diff
@@ -406,8 +430,20 @@ LRESULT ConsoleWindow::on_message(HWND wnd, UINT msg, WPARAM wp, LPARAM lp)
         if (s_windows.empty()) {
             DeleteFont(s_font);
             s_font = nullptr;
+            DeleteObject(s_brush);
+            s_brush = nullptr;
         }
         break;
+    case WM_CTLCOLORSTATIC: {
```

Comment:
Would be good to move the `WM_CTLCOLORSTATIC` case up to, say, just before `WM_ERASEBKGND` (seems like a more logical position based on the current order of the messages).